examples/wearable/UIComponents/contents/user-inputs-elements/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta content="width=device-width, initial-scale=1, user-scalable=no" name="viewport" />
+    <title>
+        Wearable UI
+    </title>
+    <link href="../../lib/tau/wearable/theme/default/tau.css" rel="stylesheet" />
+    <link href="../../lib/tau/wearable/theme/default/tau.circle.css" media="all and (-tizen-geometric-shape: circle)" rel="stylesheet" />
+    <link href="../../css/style.css" rel="stylesheet" />
+    <link href="../../css/style.circle.css" media="all and (-tizen-geometric-shape: circle)" rel="stylesheet" />
```

Comment:
media queries are no longer needed for OneUI 2.0 wearable

Reply:
Done